frontend/.nvmrc

```diff
@@ -0,0 +1 @@
+nvm use 20
```

Comment:
Is this the expected format of this file? I see this should have the version number only ([example](https://github.com/nvm-sh/nvm/issues/995)).